docs/Manuals/Program/freeciv21-client.rst

```diff
@@ -0,0 +1,151 @@
+``-H, --Hackless``
+    Do not request hack access to local, but not spawned, server.
+
+``-l, --log <FILE>``
+    Use FILE as logfile.
+
+``-M, --Meta <HOST>``
+    Connect to the metaserver at HOST.
+
+.. note::
+  With regard to the command line arguments concerning the metaserver, Freeciv21 does not have its own
+  metaserver at this time and the legacy Freeciv metaserver does not support Freeciv21. This means, right now,
+  that all commands related to the metaserver are held over from the fork from legacy Freeciv until the
+  Longturn community creates a custom metaserver.
+
+
+``-n, --name <NAME>``
+    Use NAME as username on server.
+
+``-p, --port <PORT>``
+    Connect to server port PORT (usually with ``-a``).
+
+``-P, --Plugin <PLUGIN>``
+    Use PLUGIN for sound output []. The default is `SDL`.
+
+``-r, --read <FILE>``
+    Read startup script FILE. Options are passed to the spawned server.
+
+``-s, --server <HOST>``
+    Connect to the server at HOST (usually with ``-a``).
+
+``-S, --Sound <FILE>``
+    Read sound tags from FILE. Default is `stdsounds.soundspec`.
+
+``-m, --music <FILE>``
+    Read music tags from FILE. Default is `stdmusic.musicspec`.
+
+``-t, --tiles <FILE>``
+    Use data file `FILE.tilespec` for tiles.
+
+``-w, --warnings``
+    Warn about deprecated modpack constructs.
+
+``-h, --help``
+    Displays help on command line options.
+
+``--help-all``
+    Displays help including Qt specific options.
+
+``-v, --version``
+    Displays version information.
+
+``url``
+    Server information in URL format. See examples below.
+
+EXAMPLES
+========
+
+Here are some examples:
+
+freeciv21-client -n mycoolusernname -s localhost -p 2004
+  Connect to an already running server with a username of `mycoolusername`, server host name of `localhost`,
+  and port of `2004`.
+
+freeciv21-client """fc21://[username]:[password]@[server]:[port]""" -a -t amplio2
+  Connect to an already running server using `URL` format. The parameters `[username]`, `[password]`,
+  `[server]`, and `[port]` are replaced with valid values. This command also passes the autoconnect option
+  (``-a``) as well as loads the Amplio2 Tileset.
+
+.. note::
+  The communication between the client and the server is plain text, so make sure the password you use is
+  unique. This is especially true for Longturn games on the open Internet.
+
+ENVIRONMENT
+===========
+
+The Freeciv21 client accepts these environment variables:
+
+FREECIV_DATA_PATH
+  A colon separated list of directories pointing to the Freeciv21 data directories. By default Freeciv21
+  looks in the following directories, in order, for any data files: the current directory; the "data"
+  subdirectory of the current directory; the subdirectory ".local/share/freeciv21" in the user's home
+  directory; and the directory where the files are placed by running "cmake --target install".
+
+HOME
+  Specifies the user's home directory.
+
+http_proxy
+  Set this variable accordingly when using a proxy.
```

Comment:
What does "Set this variable accordingly" mean? What should the value be?

Reply:
I think its pretty well known, but I can change the text to something more descriptive